analyzer/src/funTest/assets/projects/synthetic/spdx-project-expected-output-external-ref-with-local-ref.yml

```diff
@@ -0,0 +1,87 @@
+---
```

Comment:
Commit message nits:
- Please always use "SPDX" instead of "spdx".
- Use "Resolves #3402." instead of "Issue resolves #3402."